pkg/apis/garden/types.go

```diff
@@ -837,6 +838,10 @@ type Worker struct {
 	AutoScalerMin int
 	// AutoScalerMin is the maximum number of VMs to create.
 	AutoScalerMax int
+	// MaxSurge is maximum number of VMs that are created during an update.
+	MaxSurge intstr.IntOrString
```

Comment:
It's better to leave it as `*int64` or `*uint64`. 
https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#primitive-types

In `v1beta2` we'll change all `int` types in the public external versions to `int64`

Reply:
@mvladev can you elaborate? This can be an absolute integer value or a percentage, like in https://github.com/kubernetes/api/blob/master/extensions/v1beta1/types.go#L232-L258

Reply:
My bad, I forgot that it can be a percentage. Ignore it.

It should be `*intstr.IntOrString` then